chain/client_wrapper.sh

```diff
@@ -147,11 +147,11 @@ function parseArguments() {
       shift                # arg
       shift                # value
 
-    # Additional arguments for the Parity client.
-    # Use all remain arguments for parity.
-    elif [[ $arg == --parity-args ]] || [[ $arg == -p ]]; then
+    # Additional arguments for the client.
+    # Use all remain arguments.
+    elif [[ $arg == --client-args ]] || [[ $arg == -c ]]; then
```

Comment:
I think we will need to handle `--parity-args` and `-p` for backwards compatibility